index.bs

```diff
@@ -78,4 +84,32 @@ interface StorageBucket {
 };
 </xmp>
 
+<h3 id="storage-bucket-durability">Durability</h3>
+
+A {{StorageBucket}} has a <dfn>bucket durability</dfn>. This reflects whether the user agent
+will prioritize performance or durability when completing operations on data in the bucket.
+The value is one of the following:
+
+: "{{StorageBucketDurability/strict}}"
+:: The user agent MUST consider that operations on data in the bucket are successful only after verifying that outstanding changes have been written to a persistent storage medium.
```

Comment:
If IDB transactions are `may` should this match this as well? If the IDB transaction durability is only a hint, wouldn't this only be able to promise the same guarantee?

Reply:
The options that are passed during bucket creation may or may not be respected, however the StorageBucket property must report what the UA is actually going to do.